dmoj/cli.py

```diff
@@ -119,7 +119,7 @@ def cli_main():
         register(command(judge))
 
     def run_command(line):
-        if line[0] in commands:
+        if line and line[0] in commands:
```

Comment:
This just moves the crash to `print(ansi_style('#ansi[Unrecognized command %s](red|bold)' % line[0]))`.

Maybe you should just `if not line: return 127` in the top.

Reply:
oh yikes i didn't see that